rdm/0003-persistent-identifiers.md

```diff
@@ -0,0 +1,105 @@
+
+### External persistent identifiers
+
+#### Multiple external PIDs per record
+
+A record can have one or more external persistent identifiers (e.g. DOI, Handle, …), however a record can only have one external identifier per identifier scheme. For instance, a record can be identified by one DOI and one Handle, but cannot be identified by two DOIs. In case two DOIs exists for the same record, the problem can be resolved in one of the following manners:
+
+- Create two records, each with one DOI (will allow the DOIs to be managed).
+- Include one of the two DOIs in the record metadata as an alternative identifier for the record (means that the DOI cannot be managed by the system).
+
+#### Multiple prefixes/credentials
+
+Many globally unique identifier schemes support a notion of prefixes, and registering new identifiers in different prefixes may require different credentials. It is important that InvenioRDM is capable of handling DOIs (and other PID schemes) that have been registered with different providers. Following are examples of this situation:
+
+- One record having a DOI from DataCite, another record having a DOI from CrossRef.
+- Two records having DOIs from DataCite, but each DOI is in a different prefix requiring different credentials.
+
+#### Supported identifier schemes: DOI
+
+InvenioRDM will initially come with built-in support for DOIs as external identifiers. Further identifier schemes may be provided later. The architecture however should from the beginning allow multiple PID schemes.
+
+#### Managed vs unmanaged PIDs
+
+The external persistent identifiers can either be *managed* or *unmanaged*.
+
+- **Managed PIDs** means that it is the repository’s job to register, update and maintain the PID in an external registry.
+- **Unmanaged PIDs** means that another repository takes care of registering, updating and maintaining the PID in an external registry.
+
+This allows e.g. dataset with an existing DOI to be shared in the repository, with the repository having take control and manage the DOI.
+
+InvenioRDM must know for each external PID if it’s a managed or unmanaged DOI, since a single DOI from one prefix may be migrated to another repository (i.e. you may or may not manage all DOIs in a prefix).
+
+#### Assignment of identifiers
+A key issue in the management of persistent identifiers is when to assign which external identifier(s) for a given record.
+
+- Internal PIDs are reserved as soon as a deposit record is created.
+- External PIDs are only registered when the deposit record is published.
+
+A record must be able to explicitly defined which external PIDs should be registered with a default behaviour in case nothing is specified.
+
+#### Tombstones
+
+All persistent identifiers must support having a tombstone pages.
+
+### References
+
+- [1] Fenner (2016). Cool DOIs. https://doi.org/10.5438/55e5-t5c0
+
+## How we teach this
+
+
```

Comment:
This information should be documented/referenced in the invenio-rdm-records / invenio-pidstore.

Our grant work includes contributing writing to a "Reusable Data Best Practices Guidebook". I will translate some of the points made here to this "Guidebook" so that knowledge around identifiers becomes more widespread too.